package.json

```diff
@@ -38,6 +38,11 @@
 	},
 	"contributes": {
 		"commands": [
+			{
+				"command": "intuita.globalConfigSettings",
```

Comment:
can you rename it to `showIntuitaSettings`, these are not necessarily global settings (user vs workspace)

Reply:
addressed